example/build.gradle

```diff
@@ -30,11 +30,6 @@ android {
     }
 
     buildTypes {
-        debug {
-            minifyEnabled true
-            proguardFiles getDefaultProguardFile('proguard-android.txt')
-        }
```

Comment:
Okay thanks for the detailed explanation! I've removed the commit from my branch.